Source/WebCore/platform/graphics/BitmapImage.cpp

```diff
@@ -521,12 +517,8 @@ void BitmapImage::advanceAnimation()
     size_t nextFrame = (m_currentFrame + 1) % frameCount();
     if (!frameIsBeingDecodedAndIsCompatibleWithOptionsAtIndex(nextFrame, DecodingOptions(DecodingMode::Asynchronous)))
         internalAdvanceAnimation();
-    else {
-        // Force repaint if showDebugBackground() is on.
-        if (m_showDebugBackground)
-            imageObserver()->changedInRect(*this);
```

Comment:
Can you explain why we no longer need this?

Reply:
OK, I agree we don't want the yellow rectangle in this case.

Reply:
Since this is a behavior change, I think it is worth mentioning in the commit message.